LOGGING/src/NLogTarget/NLogTarget.csproj

```diff
@@ -53,23 +49,20 @@
     <PackageReference Include="MicroBuild.Core" Version="0.3.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="NLog" Version="4.5.11" />
```

Comment:
can you add a note about why do we need this change? Previously we were using different version for net1.3 and !net1.3 .

Reply:
Where should I write this note? The original post already says: `Updated NLog to version 4.5.11 to give access to NLog JsonSerializer for better handling of structured logging`

Reply:
My bad. I didn't read the description fully!

Reply:
No problem. NLog 4.4.12 is now 2 year old, and NLog 4.5.11 is now 1 year old. Both strong named to ver. 4.0.0.0 (with no breaking changes in NLog 4.5.11). So I think the update is very safe.